chat-plugins/mafia.js

```diff
@@ -257,6 +246,36 @@ class Mafia extends Rooms.RoomGame {
 		}
 	}
 
+	forfeit(user) {
```

Comment:
This function needs to return <code>true</code> or else you get an error message in <code>/forfeit</code>.